### ISSUE TITLE
Tables Builder Vertical style showcasing

### ISSUE THREAD

**Issue by Xanou96** [CrocoBlock/suggestions#7584]
Hi,

A vertical style tables query would help a lot. As of now only horizontal view is possible.

![5215121552](https://github.com/Crocoblock/suggestions/assets/76624963/0b1bd741-24ff-4a8e-a1ce-572de6ae0ec7)

**Comment by mechcoders** [comment]
Yes, absolutely, Vertical Table is more usable for like 2 columns......Hope so they add soon
